conformance_tests/sysman/test_sysman_frequency/src/test_sysman_frequency.cpp

```diff
@@ -408,6 +412,42 @@ TEST_F(
     }
   }
 }
+
+TEST_F(
+    FREQUENCY_TEST,
+    GivenInvalidFrequencyRangeWhenRequestingSetFrequencyThenExpectMinAndMaxFrequencyInGetFrequencyCall) {
+  for (auto device : devices) {
+    uint32_t p_count = 0;
+    auto pfreq_handles = lzt::get_freq_handles(device, p_count);
+    if (p_count == 0) {
+      FAIL() << "No handles found: "
+             << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
+    }
+
+    for (auto pfreq_handle : pfreq_handles) {
+      EXPECT_NE(nullptr, pfreq_handle);
+
+      zes_freq_range_t freq_range = {};
```

Comment:
can we have them renamed as in_freq_range and out_freq_range?

Reply:
Renamed the variables